app_dart/test/service/luci_test.dart

```diff
@@ -0,0 +1,22 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:cocoon_service/src/service/luci.dart';
+
+import 'package:test/test.dart';
+
+void main() {
```

Comment:
Did we forget the real change? seems like only tests were uploaded

Reply:
Could you refresh the page. The main change is in the first commit.